### PR TITLE
JC-2086 It's impossible to refresh CAPTCHA via Tab

### PR DESCRIPTION
**Pull request by SurfVaporizer**
- Can't move focus to "Captcha text" field via Tab in Firefox - fixed

### CODE REVIEW

File: jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 <div class="container form-login-related registration-page">
   <form:form id="form" name="form" action='${pageContext.request.contextPath}/user/new'
-             modelAttribute="newUser" method="POST" class='form-vertical'>
+             modelAttribute="newUser" method="POST" class='form-vertical' tabindex="-1">
```
Comment (masyan) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: зачем это тут? у нас порядок фокуса по табу определяется в js генерирующем диалоговое окно

Comment (SurfVaporizer) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: на самом деле можно убрать, я не мог разобраться почему не фокус по табу слетает в FF.

Comment (ctapobep) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: So AFAIU the focus order has issues in FF, @masyan maybe you could take a look and help @SurfVaporizer with that? 
@SurfVaporizer did I understand the situation correctly?

Comment (SurfVaporizer) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: @ctapobep now the focus issue is solved in FF. Focus work correctly with tabindex param in the form or without it.

Comment (ctapobep) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: So, @masyan would you be able to have a look why our dialog JS doesn't work for FF in this case?

Comment (masyan) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: it works. on my ff 41.0.2/linux/preprod

Comment (ctapobep) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: Try on QA, PREPROD has an old version. Make sure you're trying the page, not the dialog.

**Comment by masyan** [comment]
works fine. (QA)

File: jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp (hunk 1 above)
Comment (masyan) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: works fine. (QA)

Comment (ctapobep) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp]: That's very weird.. I can reproduce the issue on QA with my FF 41.0.2. Maybe there is a difference between FFs on different platform though that's still weird. 

So given the bug is reproduced on others' machines, do you think that current fix is applicable?

**Comment by ctapobep** [comment]
This has already been merged, right?

**Pull request closed without merging**